### ISSUE TITLE
Couldn't install without specifying an older version of scikit-image (0.17.2)

### ISSUE THREAD

**Issue by ryandikdan** [spacetx/starfish#1988]
Hello, I have been trying to install starfish and have come across an error. I couldn't import starfish after installing it, although installing doesn't throw errors really.

I've made either conda environments or python environments in python 3.6, 3.7, 3.8, and 3.10
`conda create -n starfish python=3.x`
 then run `pip install starfish`, and in python of ipython when I run `import starfish` or any other variation it tries to load it, but then throws the error: 
`ImportError: cannot import name 'register_translation' from 'skimage.feature'`
then lists where the skimage __init__.py is.

I've tried to create a new environment and install numpy first and there was no difference. Looks like it's a problem with skimage. 

I finally resolved the issue by running 
`pip install scikit-image==0.17.2`
after installing starfish, since it seems that in order to be imported, starfish requires the now deprecated function 'register_translation' from 'skimage.feature' as listed in the error message. Could you replace the deprecated function or remove it so that other people don't have as much an issue with installation? Or perhaps add that you need to install that specific version of scikit-image?

**Comment by ryandikdan** [comment]
Actually, when trying to run the quick tutorial, exactly that function threw an error also. Perhaps this is something you're working out.

**Comment by berl** [comment]
hi @ryandikdan  this has been fixed for a while, but we haven't made a `pypi` release with the the updates. You should be able to clone the repo and install following the instructions [here](https://spacetx-starfish.readthedocs.io/en/latest/developer_guide/)

**Comment by berl** [comment]
I'm going to close this, but feel free to reopen if installing from the repo doesn't work.

**Comment by ryandikdan** [comment]
I did get it to work, although there were some issues. I had to install numpy and napari before running `make install-dev`. To download it I changed the git:// to https:// compared to the website recommendation. Also for WSL I had to download a server Xming, and update to Windows 11 for WSLg. Otherwise the only issue running the quick start tutorial was that it said the watershed function from scikit-image was being deprecated, but it worked. Hope this helps!
